src/tests/toml-invalid-cfg/Cargo.toml

```diff
@@ -1,2 +1,5 @@
+[package]
+name = "toml-invalid-cfg"
+
 [package.metadata.system-deps.'cfg(badger)']
-testanotherlib = { version = "1", optional = true }
```

Comment:
I think you can leave those out from the patch.

Reply:
Sorry about that, not sure what happened there, it probably was the editor autoformatting something when saving the file. I removed the changes from the diff :)